readthedocs/projects/signals.py

```diff
@@ -34,5 +35,12 @@ def handle_project_import(sender, **kwargs):
             if service is not None:
                 if service.setup_webhook(project):
                     messages.success(request, _('Webhook activated'))
+                    _set = True
                 else:
                     messages.error(request, _('Webhook configuration failed'))
+    if not _set:
+        messages.error(
+            request,
+            _('No accounts available to set webhook on. '
+                'Please connect either a GitHub or BitBucket account.')
```

Comment:
Couple issues:
* indentation is off on line 45
* `BitBucket` -> `Bitbucket`
* if the project being imported was at GitHub, we don't want to tell the user to connect a Bitbucket account. we have the service provider above, I feel like that should be used here to give the user a more guided error.